src/components/sidebar/right-sidebar/RightSidebar.css

```diff
@@ -3,7 +3,8 @@
 }
 
 .filters-container {
-    overflow: auto;
+    overflow-y: auto;
```

Comment:
WRT overflow-y: do we want to expand the size of this box? When you click on a dropdown filter (e.g., population group), a vertical scrollbar appears, presumably because the height of the filters container is limited

Reply:
Do you mean this 
![image](https://user-images.githubusercontent.com/21212898/82245504-0a07d280-9911-11ea-8f13-84a7feaf6942.png)

I think that's just default behaviour for the dropdown ui component we are using. The overflow-y just allows the whole filters section to be scrollable if the browser isn't tall enough (and I removed the scrollbar for aesthetic reasons there but we can readd it).

As for changing the dropdown component's behaviour, we can make it expand it fit, however I'm not sure that's the best user experience. Let's say the countries dropdown has like 50 options sometime in the future. If we were to expand the size of the dropdown box, it would take > the browser height to show the expanded dropdown, which seems a little weird.